x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -60,25 +60,26 @@ For example, using the `005010` version and the 850 transaction set, the schema
 
 If you're using one or more custom schemas, refer to xref:x12-edi-connector-config-topics.adoc[XML Schema and Trading Partner Configuration] for complete information on setting your schema locations.
 
-== About X12 Connector Message Structure
+== About X12 EDI Connector Message Structure
 
 This connector enables the reading or writing of X12 documents into (or from) the canonical EDI message structure.
```

Comment:
Fixed. It is and probably not necessary. I think the PM wants to strike all of this content in a future enhancement because its not to the level of this user.